platform/darwin/src/MGLMapSnapshotter.h

```diff
@@ -12,40 +12,46 @@ MGL_EXPORT
 @interface MGLMapSnapshotOptions : NSObject
 
 /**
- Creates a set of options with the minimum required information
- @param styleURL the style url to use
- @param camera the camera settings
- @param size the image size
+ Creates a set of options with the minimum required information.
+ 
+ @param styleURL URL of the map style to snapshot. The URL may be a full HTTP or HTTPS URL,
+ a Mapbox URL indicating the style’s map ID (mapbox://styles/{user}/{style}), or a path
+ to a local file relative to the application’s resource path. Specify nil for the default style.
+ @param size The image size.
  */
-- (instancetype)initWithStyleURL:(NSURL*)styleURL camera:(MGLMapCamera*)camera size:(CGSize)size;
+- (instancetype)initWithStyleURL:(nullable NSURL*)styleURL camera:(MGLMapCamera*)camera size:(CGSize)size;
 
-#pragma mark - Configuring the map
+#pragma mark - Configuring the Map
 
 /**
-  The style URL for these options.
+ URL of the map style to snapshot. The URL may be a full HTTP or HTTPS URL,
+ a Mapbox URL indicating the style’s map ID (mapbox://styles/{user}/{style}), or a path
+ to a local file relative to the application’s resource path. Specify nil for the default style.
```

Comment:
This property is read-only, so it isn’t possible for the developer to specify anything. In fact, if the developer passes `nil` into `-initWithStyleURL:camera:size:`, `styleURL` ends up non-nil because we choose a default style.